posts/project-pythia-overview.md

```diff
@@ -0,0 +1,76 @@
+
+Clicking on the link on the Pythia Portal will take you to the JupyterBook containing the instructional material - this can also be accessed using [foundations.projectpythia.org](https://foundations.projectpythia.org/landing-page.html).
+
+![foundations link](images/pythia_foundations_link.png)
+
+Once within the [Foundations Book](https://foundations.projectpythia.org/landing-page.html), you will see the typical [Jupyterbook](https://jupyterbook.org/intro.html) interface with a few primary sections.
+
+![foundations sections](images/foundations_diagram.png)
+
+Each notebook is its own "page", separated into sections outlined in the table of contents which can be found along the left sidebar. On the right side of the page is the table of contents for a single notebook, which is determined using the header information in each notebook. This can be useful when wanting to navigate to specific section of a notebook.
+
+These notebooks are executable - which is what makes this [Jupyterbook](https://jupyterbook.org/intro.html) interface so useful! If you select the space ship button near the top of the page, the notebook will open up on [binder](https://mybinder.org/) which enables you to interactively run the Jupyter Notebook on the cloud.
+
+If you would like to download a local copy of the notebook, select the download button in the top right corner, and select whether to download as a `PDF` or `ipynb` (notebook).
+
+![pandas example](images/pythia_pandas.png)
+
+Within the foundational content notebooks, you will find that each notebook follows the same format, including the following sections:
+
+- **Overview** - a brief overview of what is covered in that notebook
+- **Prerequisites** - what material is required before diving into the content
+- **Desired experience level** - the difficulty level of the notebook
+- **Time to learn** - estimated time to complete notebook
+- _The actual content_ - which often includes imports, and several headings (ex. `The Pandas Dataframe`)
+- **Summary** - main takeaways from the notebook
+- **What's next** - what is covered in additional notebooks in that section
+- **Resources and references** - data references, where to find more resources
+
+#### [Resource Gallery](https://projectpythia.org/gallery.html)
+
+The [Resource Gallery](https://projectpythia.org/gallery.html) includes links to a variety of external (to Project Pythia) material that is useful to the Geoscience community! This is a great place to go if you are looking for additional training or reference materials or more specific examples or galleries.
+
+![pandas example](images/pythia_resource_gallery.png)
+
+If you continue scrolling down the page, you will find various ways to contribute and engage with the project, including via [Github Discussions](https://github.com/ProjectPythia/projectpythia.github.io/discussions) and by [contributing](https://projectpythia.org/contributing.html)!
```

Comment:
"you'll find various ways to contribute ...and by contributing" 

I think we need to use slightly different language here since the first "contribute" is more vague and the second should be more specific. 

Do you want this section to be about contributing in general or specifically to the research gallery?

Reply:
See #86